README.md

```diff
@@ -16,20 +16,142 @@ Capabilities provided by `caikit-nlp`:
 | Tokenization         | 1. `RegexSentenceSplitter`                | 1. Demo purposes only                                                                                                                                       |
 | Embedding         | [COMING SOON]                | [COMING SOON]                                                                                                                                       |
 
-### Getting Started
+## Getting Started
+
+### Notebooks
 
 To help you quickly get started with using Caikit, we have prepared a [Jupyter notebook](examples/Caikit_Getting_Started.ipynb) that can be run in Google Colab. Caikit-nlp is a powerful library that leverages prompt tuning and fine-tuning to add NLP domain capabilities to caikit.
 
+### Installation
+
+To install from git repo:
+
+```bash
+python -m venv .venv
+source .venv/bin/activate
+pip install git+https://github.com/caikit/caikit-nlp
+```
+
+### Bootstrapping models
+
+`caikit_nlp` can use  Hugging Face models, allowing for direct download and bootstrapping.
+
+For example, to use [google/flan-t5-small](https://huggingface.co/google/flan-t5-small):
+
+```python
+import os
+# The env var ALLOW_DOWNLOADS has to be set to allow model downloads before importing caikit_nlp
+os.environ['ALLOW_DOWNLOADS'] = "1"
```

Comment:
there are some other ways to configure this as well, like via config file, but I think keeping this referring to environment variable is more helpful that way a user instantly knows that they can just configure this via environment variable. No change needed, but just mentioning it here.